.github/workflows/test-deploy.yml

```
@@ -4,23 +4,27 @@ on:
     - cron: 0 0 * * *
   release:
     types: [published]
-  pull_request:
+  pull_request_target:
 jobs:
   lint:
     runs-on: ubuntu-latest
```

```suggestion
  lint:
    environment:
      ${{ (github.event_name == 'pull_request_target' &&
      github.event.pull_request.head.repo.full_name != github.repository) &&
      'test-external' || 'test-internal' }}
    runs-on: ubuntu-latest
```
Running `npm` on untrusted code is also dangerous. Let's protect the lint job with the environment.